### PR TITLE
various fixes to this to use the load_config and siteType from the co…

### PR DESCRIPTION
**Pull request by mhines-usgs**
…nfig

### CODE REVIEW

File: scripts/getSiteData.R [hunk 1]
```diff
@@ -1,41 +1,42 @@
...
-    sites$id[i] <- prmnn_i
+    wqp_sites$id[i] <- prmnn_i
     print(i)
   }
-  write.csv(sites, file=paste0(getwd(),"/data/wqp_nhd/sitesPermId",states[[j]]$fips,".csv"),row.names=FALSE) 
+  write.csv(wqp_sites, file=paste0("data/wqp_nhd/sitesPermId",states[[j]]$fips,".csv"),row.names=FALSE) 
 }
 
 #join all the site files for use as a lookup table
 lookup <- data.frame()
```
Comment (mhines-usgs) [inline scripts/getSiteData.R:36]: @jread-usgs is this where you're suggesting i use the stringsAsFactors=FALSE ?

Comment (jordansread) [inline scripts/getSiteData.R:36]: yes, 
`lookup <- data.frame(stringsAsFactors=FALSE)`

Comment (mhines-usgs) [inline scripts/getSiteData.R:36]: ok, that's what I thought, but when I opened the rds file back up after saving it, it looks like this:

image

**Comment by jordansread** [comment]
ok, no. whhhhoooops

**Pull request merged**